BitwardenShared/UI/Platform/Application/AppProcessor.swift

```diff
@@ -169,7 +169,7 @@ public class AppProcessor {
 
         // Check for specific URL components that you need.
         guard let params = components.queryItems,
-              let host = components.host else {
+              components.host != nil else {
```

Comment:
🤔 Do we even need to check the `host` here?

Reply:
if it does not have a host, it should be a good reason not to continue 🤔